metrics/ddl.go

```diff
@@ -109,7 +109,8 @@ const (
 	LblVersion = "version"
 )
 
-func init() {
+// RegisterMetrics registers the metrics which are ONLY used in TiDB server.
+func RegisterMetrics() {
```

Comment:
Not only this init function, there are many init metrics function in tidb, you can grep `MustRegister` to find all of them.